pages/blog/index.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <b-container data-qa="blog">
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <h1>{{ $t('blog.blogs') }}</h1>
+      </b-col>
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          data-qa="blog posts"
+        >
+          <ContentCard
+            v-for="(post, index) in posts"
+            :key="index"
+            :title="post.fields.headline"
+            :url="{ name: 'blog-all', params: { pathMatch: post.fields.identifier } }"
+            image-url="https://picsum.photos/600/300/?image=25"
```

Comment:
Can this be replaced by an image from Contentful, since the work on EC-3918 was completed?

Reply:
Im looking at https://app.contentful.com/spaces/i01duvb6kq77/environments/test/entries/XyvOiNMv0yUY5EKal4NcT I can not see a field for images at present

Reply:
It should be there now.